packages/tools/test/npm.coffee

```diff
@@ -0,0 +1,79 @@
+
+nikita = require '@nikitajs/tools'
+{tags, ssh} = require './test'
+they = require('ssh2-they').configure ssh...
+
+return unless tags.tools_npm
+
+describe 'tools.npm', ->
+
+  describe 'schema', ->
+
+    they 'name is required', ({ssh}) ->
+      nikita
+        ssh: ssh
+      .tools.npm
+        global: true
+      , (err, {status}) ->
+        status.should.be.false() unless err
```

Comment:
This should throw an error, no ? I would expect you to check the error message instead of the status which is not related to this concern.